R/router.R

```diff
@@ -203,9 +205,36 @@ create_router_callback <- function(root, routes = NULL) {
 #'     )
 #'   }
 #' }
+#' \dontrun{
```

Comment:
Please add a minimal description to this example.

Reply:
Added few comments